### ISSUE TITLE
Checkboxes having 3 dots when not checked when grid is in multi selection mode

### ISSUE THREAD

**Issue by Klaudeta** [vaadin/vaadin-grid-flow#552]
Three dots present when checkbox not selected:

<img width="768" alt="screenshot 2019-02-22 at 10 48 08" src="https://user-images.githubusercontent.com/31614592/53230918-82d99980-3690-11e9-8c91-dfbc4f242ea4.png">

Issue present only with the Flow Grid

**Comment by jouni** [comment]
This is caused by the unnecessary white-space characters in the [selection column template](https://github.com/vaadin/vaadin-grid-flow/blob/master/vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/vaadin-grid-flow-selection-column.html):

For example, this:
```html
<vaadin-checkbox
        id="selectAllCheckbox"
        aria-label="Select All"
        hidden$="[[selectAllHidden]]"
        on-click="_onSelectAllClick"
        checked="[[selectAll]]">
      </vaadin-checkbox>
```
Should be:
```html
<vaadin-checkbox
        id="selectAllCheckbox"
        aria-label="Select All"
        hidden$="[[selectAllHidden]]"
        on-click="_onSelectAllClick"
        checked="[[selectAll]]"></vaadin-checkbox>
```

**Comment by jouni** [comment]
Here’s the change that caused this as a regression for the Flow grid: https://github.com/vaadin/vaadin-grid/commit/bbd821b89b71652c38d2d47f9f2da73b19dc446d#diff-d1022b74395ce930104a7f92625afb06

**Comment by ujoni** [comment]
Is still issue still present in master?
There was a PR https://github.com/vaadin/vaadin-grid-flow/pull/520 which attempted to address this issue before.

**Comment by heruan** [comment]
Hm now I see how #520 was handled: using `px` is a strong choice, what about having set a smaller/bigger font-size? Checkboxes adapt to the font-size (they use `em`).

**Comment by jouni** [comment]
> Is still issue still present in master?
> There was a PR #520 which attempted to address this issue before.

Yes, still present in master – see the source code link in my first comment.

The fix is to remove the white-space characters within the `<vaadin-checkbox></vaadin-checkbox>` elements in the selection column template.

**Comment by jouni** [comment]
@heruan, yeah, using a fixed pixel size is a poor workaround. Using `em` would indeed be better. The real solution is to have the column content be based on the content size (which is in the backlog for the web component).

**Comment by integrigy** [comment]
Encountered this issue with Chrome whenever having a component in a grid including buttons as well as ComboBox or Select in header for filtering.  To work-around added the following to shared-styles.html -

```
<dom-module id="my-grid" theme-for="vaadin-grid">
   <template>
      <style>
	  
      [part~="cell"] ::slotted(vaadin-grid-cell-content) {
        text-overflow: clip;
      }      
     
      </style>
   </template>
</dom-module>
```
As @jouni referenced above [vaadin/vaadin-grid@bbd821b#diff-d1022b74395ce930104a7f92625afb06](https://github.com/vaadin/vaadin-grid/commit/bbd821b89b71652c38d2d47f9f2da73b19dc446d#diff-d1022b74395ce930104a7f92625afb06), the problem was introduced with the `text-overflow: ellipsis;`.

**Comment by web-padawan** [comment]
Should be fixed by #868 which makes that column adapt to width of checkbox. Closing.

**Comment by InnaSnytkina** [comment]
Hi,

I'm using Vaadin 8.11.0. I see checkbox with 2 dots in EDGE, Chrome, Opera browsers when for the Grid the SelectionMode is MULTI. In Mozilla looks good, without dots. 
Any advise how to fix or workaround that issue?
![image](https://user-images.githubusercontent.com/43446807/85756680-b65e8500-b717-11ea-8a89-0d9ddc77475d.png)
